strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -27,198 +27,112 @@
 
     def prepare_thermal_state(self, nbar, mode):
-        """
-        Prepare the vacuum state on the specified mode.
-
-        Note: this may convert the state representation to mixed.
-
-        Args:
-            nbar (float): mean number of photons in the mode
-            mode (int): index of mode where state is prepared
-        """
         self.circuit.init_thermal(nbar, mode)
 
     def prepare_vacuum_state(self, mode):
-        """
-        Prepare the vacuum state on the specified mode.
-
-        Note: this may convert the state representation to mixed.
-
-        Args:
-            mode (int): index of mode where state is prepared
-        """
         self.circuit.loss(0.0, mode)
 
     def prepare_coherent_state(self, alpha, mode):
-        """
-        Prepare a coherent state with parameter alpha on the specified mode.
-
-        Args:
-            alpha (complex): coherent state displacement parameter
-            mode (int): index of mode where state is prepared
-        """
-
         self.circuit.loss(0.0, mode)
         self.circuit.displace(alpha, mode)
 
     def prepare_squeezed_state(self, r, phi, mode):
-        """
-        Prepare a coherent state with parameters (r, phi) on the specified mode.
-
-        Args:
-            r (float): squeezing amplitude
-            phi (float): squeezing phase
-            mode (int): index of mode where state is prepared
-        """
         self.circuit.loss(0.0, mode)
         self.circuit.squeeze(r, phi, mode)
 
     def rotation(self, phi, mode):
-        """
-        Perform a phase-space rotation by angle phi on the specified mode.
-
-        Args:
-            phi (float):
-            mode (int): index of mode where operation is carried out
-        """
         self.circuit.phase_shift(phi, mode)
 
     def displacement(self, alpha, mode):
-        """
-        Perform a displacement operation on the specified mode.
-
-        Args:
-            alpha (float): displacement parameter
-            mode (int): index of mode where operation is carried out
-        """
         self.circuit.displace(alpha, mode)
 
     def squeeze(self, z, mode):
-        """
-        Perform a squeezing operation on the specified mode.
-
-        Args:
-            z (complex): squeezing parameter
-            mode (int): index of mode where operation is carried out
-
-        """
         phi = angle(z)
         r = abs(z)
         self.circuit.squeeze(r, phi, mode)
 
     def prepare_displaced_squeezed_state(self, alpha, r, phi, mode):
-        """
-        Prepare a displaced squezed state with parameters (alpha, r, phi) on the specified mode.
-        Note: this may convert the state representation to mixed.
-
-        Args:
-            alpha (complex): displacement parameter
-            r (float): squeezing amplitude
-            phi (float): squeezing phase
-            mode (int): index of mode where state is prepared
-        """
-
+        # FIXME reset to vacuum missing!
```

Comment:
Let's do this in this PR, or make a new issue for it

Reply:
I'll just add the reset there, that should fix it.